azext_iot/_help.py

```diff
@@ -633,6 +633,61 @@
 """
 '''
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Manage the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Export the state of an IoT Hub to a file.
+    long-summary: The exported state will include hub configurations, edge deployments, device identities and twins, module
+                  identities and twins, hub system and user identities, certificates, endpoints and routes.
+    examples:
+    - name: Export the state of the specified hub to the specified file.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename}
+    - name: Export the state of the specified hub to the specified file, overwriting the file contents.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename} --force
+"""
+
+helps[
+    "iot hub state import"
+] = """
+    type: command
+    short-summary: Import a Hub state from a file to an IoT Hub.
+    long-summary: The imported state will include hub configurations, edge deployments, device identities and twins, module
+                  identities and twins, hub system and user identities, certificates, endpoints and routes.
+    examples:
+    - name: Import the state from the specified file to the specified hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename}
+    - name: Import the state from the default file to the specified hub, replacing the previous state of the hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename} --replace
+"""
+
+helps[
+    "iot hub state migrate"
+] = """
+    type: command
+    short-summary: Copy the state of one hub to another hub without saving to a file.
```

Comment:
If the term `migrate` is preferred to `copy` we should replace the first word here.